packages/orbit-components/src/InputField/InputField.stories.tsx

```diff
@@ -10,236 +11,238 @@ import ServiceLogo from "../ServiceLogo";
-};
-
-export const DefaultInput = ({ label, value, placeholder }) => {
-  return (
-    <InputField label={label} value={value} placeholder={placeholder} onChange={action("change")} />
-  );
+type InputFieldPropsAndCustomArgs = React.ComponentProps<typeof InputField> & {
+  serviceLogoName?: Name;
+  grayScale?: boolean;
 };
 
-DefaultInput.story = {
-  name: "Default input",
+const meta: Meta<InputFieldPropsAndCustomArgs> = {
+  title: "InputField",
+  component: InputField,
 
   parameters: {
-    info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: ["onChange"],
+    },
   },
-};
 
-DefaultInput.args = {
-  label: "Label",
-  value: "",
-  placeholder: "Placeholder",
-};
-
-export const NumberInput = ({ label, value, placeholder, maxValue, minValue }) => {
-  return (
-    <InputField
-      type="number"
-      value={value}
-      label={label}
-      placeholder={placeholder}
-      maxValue={maxValue}
-      minValue={minValue}
-      onChange={action("change")}
-    />
-  );
+  args: {
+    label: "Label",
+    value: "",
+    placeholder: "Placeholder",
+    help: "",
+    error: "",
+    onChange: action("change"),
+    required: false,
+    inlineLabel: false,
+    type: TYPE_OPTIONS.TEXT,
+    disabled: false,
+    readOnly: false,
+    helpClosable: true,
+  },
+
+  argTypes: {
+    type: {
+      options: Object.values(TYPE_OPTIONS),
+      control: {
+        type: "select",
+      },
+    },
+    prefix: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
+    },
+    suffix: {
+      options: Object.keys(Icons),
+      control: {
+        type: "select",
+      },
+    },
+    spaceAfter: {
+      options: Object.values(SPACINGS_AFTER),
+      control: {
+        type: "select",
+      },
+    },
+  },
 };
 
-NumberInput.story = {
-  name: "Number input",
+export default meta;
+type Story = StoryObj<InputFieldPropsAndCustomArgs>;
 
+export const DefaultInput: Story = {
   parameters: {
-    info: "Some description about this type of InputField in general.",
+    controls: {
+      exclude: [
+        "help",
+        "error",
+        "onChange",
+        "required",
+        "inlineLabel",
+        "type",
+        "disabled",
+        "readOnly",
+        "spaceAfter",
+        "prefix",
+        "suffix",
+        "helpClosable",
+      ],
+    },
   },
-};
 
-NumberInput.args = {
-  label: "Number",
-  value: "2",
-  placeholder: "Number",
-  maxValue: 3,
-  minValue: 1,
-};
-
-export const PasswordInput = ({ label, value, placeholder }) => {
-  return (
-    <InputField
-      type="password"
-      label={label}
-      value={value}
-      placeholder={placeholder}
-      onChange={action("change")}
-    />
-  );
+  args: {
+    help: "",
+    error: "",
+    required: false,
+    inlineLabel: false,
+    type: TYPE_OPTIONS.TEXT,
+  },
 };
 
-PasswordInput.story = {
-  name: "Password input",
-
+export const NumberInput: Story = {
```

Comment:
I rechecked our reports and this was not part of it. As it it related to Label, maybe it could be fixed as part of this task? https://kiwicom.atlassian.net/browse/FEPLT-2098

Reply:
Ok, please add it to the Jira ticket.

Reply:
Added to the ticket ✅